### PR TITLE
[flexcounter]: add support to change port/queue counter poll interval

### PR DESCRIPTION
**Pull request by sihuihan88**
Signed-off-by: Sihui Han <sihan@microsoft.com>

**What I did**
Add support to change port/queue counter poll interval
**Why I did it**

**How I verified it**
Tested on DUT.
**Details if related**

### CODE REVIEW

File: orchagent/flexcounterorch.cpp [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        auto data = kfvFieldsValues(t);
+
+        if (!flexCounterGroupMap.count(key))
+        {
+            SWSS_LOG_NOTICE("Invalid flex counter group input, %s", key.c_str());
+            consumer.m_toSync.erase(it++);
+            continue;
+        }
+
+        if (op == SET_COMMAND)
```
Comment (marian-pritsak) [inline orchagent/flexcounterorch.cpp]: IFAIK if nothing is set, the default value of 1 second is used.
Then I think we also have to set the default back if the entry is deleted from DB.

**Comment by sihuihan88** [comment]
retest this please

File: orchagent/flexcounterorch.cpp (hunk 1 above)
Comment (sihuihan88) [reply orchagent/flexcounterorch.cpp]: This entry only has set_command and no del command. We can always use set command to change the interval value.

**Comment by lguohan** [comment]
retest this please

File: orchagent/flexcounterorch.cpp (hunk 1 above)
Comment (marian-pritsak) [reply orchagent/flexcounterorch.cpp]: ok

**Pull request merged**